miden-lib/src/tests/test_prologue.rs

```diff
@@ -193,18 +193,10 @@ fn chain_mmr_memory_assertions<A: AdviceProvider>(
     // The number of leaves should be stored at the CHAIN_MMR_NUM_LEAVES_PTR
     assert_eq!(
         process.get_mem_value(ContextId::root(), CHAIN_MMR_NUM_LEAVES_PTR).unwrap()[0],
-        Felt::new(inputs.block_chain().mmr().forest() as u64)
+        Felt::new(inputs.block_chain().num_block() as u64)
     );
 
-    for (i, peak) in inputs
-        .block_chain()
-        .mmr()
-        .peaks(inputs.block_chain().mmr().forest())
-        .unwrap()
-        .peaks()
-        .iter()
-        .enumerate()
-    {
+    for (i, peak) in inputs.block_chain().peaks().peaks().iter().enumerate() {
```

Comment:
The `.peaks().peaks()` here is unfortunate, and it probably should be something like `.peaks().as_digests()`, but that would be a breaking change (affecting Miden VM as well) - so, I left it as is for now.